### PR TITLE
Trim out dogfood constants. Comment out currently unused device login codes

### PR DESCRIPTION
**Pull request by shhsu**
Major clean up

### CODE REVIEW

**Comment by shhsu** [comment]
@sajaya @DavidObando

**Comment by sajayantony** [comment]
Please remove commented code.

File: src/docker-credential-acr/aad_prod.go [hunk 1]
```diff
@@ -1,6 +0,0 @@
-// +build !dogfood
-
-package main
-
-const constAADServer = "login.microsoftonline.com"
```
Comment (DavidObando) [inline src/docker-credential-acr/aad_prod.go:5]: Is this not needed to renew?

Comment (shhsu) [reply src/docker-credential-acr/aad_prod.go:5]: nope, renew calls ACR exchange endpoint only.

**Pull request merged**